### PR TITLE
release automation

### PR DESCRIPTION
**Pull request by intoeetive**

### CODE REVIEW

File: system/ee/ExpressionEngine/Cli/Commands/Upgrade/UpgradeMap.php [hunk 1]
```diff
@@ -153,10 +154,6 @@ class UpgradeMap
     ];
 
     public static $versionNaming = [
-        '6.0.3' => '6.0.3',
```
Comment (TomJaeger) [inline system/ee/ExpressionEngine/Cli/Commands/Upgrade/UpgradeMap.php:156]: @intoeetive Why were these removed?

Comment (intoeetive) [reply system/ee/ExpressionEngine/Cli/Commands/Upgrade/UpgradeMap.php:156]: This array holds info for versions where version name does not match version number. If they are same, they can/should be omited (you can inspect the code to ensure)

File: system/ee/ExpressionEngine/Tests/.phpunit.result.cache [hunk 2]
```diff
@@ -0,0 +1 @@
...dation\ValidatorTest::testLimitHtml with data set #8";d:0;s:87:"ExpressionEngine\Tests\Service\Validation\ValidatorTest::testLimitHtml with data set #9";d:0;s:88:"ExpressionEngine\Tests\Service\Validation\ValidatorTest::testLimitHtml with data set #10";d:0;s:88:"ExpressionEngine\Tests\Service\Validation\ValidatorTest::testLimitHtml with data set #11";d:0;s:88:"ExpressionEngine\Tests\Service\Validation\ValidatorTest::testLimitHtml with data set #12";d:0;s:88:"ExpressionEngine\Tests\Service\Validation\ValidatorTest::testLimitHtml with data set #13";d:0;s:43:"LanguageKeysTest::testDuplicateLanguageKeys";d:0;s:45:"LanguageKeysTest::testDuplicateLanguageValues";d:0.002;s:54:"LanguageKeysTest::testDuplicateLanguageKeysAcrossFiles";d:0.007;s:40:"LanguageKeysTest::testUnusedLanguageKeys";d:0.002;s:29:"TypographyTest::testCodeFence";d:0.087;s:29:"TypographyTest::testCodeBlock";d:0.001;s:37:"TypographyTest::testCodeBlockAndFence";d:0.007;s:31:"TypographyTest::testSmartyPants";d:0.003;s:28:"TypographyTest::testNoMarkup";d:0.068;s:35:"TypographyTest::testLinksWithSpaces";d:0.018;s:40:"TypographyTest::testEmoticonConversionOn";d:0.001;s:41:"TypographyTest::testEmoticonConversionOff";d:0.001;}}}
```
Comment (TomJaeger) [inline system/ee/ExpressionEngine/Tests/.phpunit.result.cache]: @intoeetive Should this file be in the repo

Comment (intoeetive) [reply system/ee/ExpressionEngine/Tests/.phpunit.result.cache]: No. Pushed commit to removed and gitignore.

File: tests/cypress/admin.cypress.json [hunk 3]
```diff
@@ -9,7 +9,7 @@
     "requestTimeout": 20000,
     "watchForFileChanges": false,
     "chromeWebSecurity": false,
-    "testFiles": "@(utilities|site|settings|logs|jumps|field|channel|addons)/*.ee6.js",
+    "testFiles": "@(settings|addons)/*.ee6.js",
```
Comment (TomJaeger) [inline tests/cypress/admin.cypress.json:12]: @intoeetive where these other tests removed and placed back in somewhere else to make more go in parallel?

**Review by TomJaeger — CHANGES_REQUESTED** [review]
@intoeetive Looking for a little feedback from you on a few items in here.

File: tests/cypress/admin.cypress.json (hunk 3 above)
Comment (intoeetive) [reply tests/cypress/admin.cypress.json:12]: @TomJaeger that's right

**Review by matthewjohns0n — COMMENTED** [review]
I reviewed the build files. Looks good to me.

**Review by TomJaeger — APPROVED** [review]
